G/GracePlot/Deps.toml

```diff
@@ -0,0 +1,3 @@
+["0.3.2-0"]
+Printf = "de0858da-6303-5e67-8744-51eddeeeb8d7"
+Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
```

Comment:
You don't seem to be using Printf, and Test can maybe be a test-only dependency?

Reply:
Yes.  Thought I was using Printf in most units - but I don't (blindly copying).  I will remove it in the future.

Also: I don't use the macros yet from the packages I just registered (yet).  I guess I will remove the entries in future commits.

HOWEVER: Test is ***typically*** a test-only dependency.  How do I add a test-only dependency?  Is there documentation on this?

Reply:
https://julialang.github.io/Pkg.jl/v1/creating-packages/#Test-specific-dependencies-1

Reply:
Thanks:
  - Fixed master branches of all modules (will not tag new release unless requested).
  - NOTE: Could not find doc for multiple deps in `test` target.  Had to use `Colors.jl` as an example.

Reply:
Would be a nice contribution to the Pkg manual if you want to add it.